config.py

```diff
@@ -96,7 +95,7 @@
 pv["technical"] = {"T_r": 25,
                    "loss_coeff": 0.37,
                    "tracking": 0,
-                   "orientation": 180  # 0: South; 90: West ; 180: North; -90: East
+                   "orientation": 0  # | 0: South | 90: West | 180: North | -90: East |
```

Comment:
Added a legend for the orientation, i'm not sure if the east and west are correct, to be validated